internal/acctest/acctest.go

```diff
@@ -408,8 +408,9 @@ func MatchResourceAttrGlobalHostname(resourceName, attributeName, serviceName st
 // CheckResourceAttrGlobalARN ensures the Terraform state exactly matches a formatted ARN without region
 func CheckResourceAttrGlobalARN(resourceName, attributeName, arnService, arnResource string) resource.TestCheckFunc {
 	return func(s *terraform.State) error {
+		accountID := strings.Split(AccountID(), "@")[1]
```

Comment:
```suggestion
		customerName := strings.Split(AccountID(), "@")[1]
```

Так понятнее что оттуда достается.

Еще я бы тут все таки проверяла, что `@` присутствует в строке. Можно сделать отдельную функцию `CustomerName()`, в которой проверки можно будет спрятать.